omisego-sdk/src/test/kotlin/co/omisego/omisego/OMGAPIClientTest.kt

```diff
@@ -161,7 +168,7 @@ class OMGAPIClientTest {
         val callback: Callback<OMGResponse<User>> = mock()
         omgAPIClient.getCurrentUser().enqueue(callback)
 
-        val apiError = APIError(ErrorCode.SERVER_INTERNAL_SERVER_ERROR, "The EWallet API was 500 Internal Server Error")
+        val apiError = co.omisego.omisego.model.APIError(ErrorCode.SERVER_INTERNAL_SERVER_ERROR, "The EWallet API was 500 Internal Server Error")
```

Comment:
Why FQCN?

Reply:
There's something wrong with my Android Studio's auto import 😂. Fixed it

Reply:
This can sometimes happen when you move/rename some classes :/